src/azure-cli/azure/cli/command_modules/servicebus/commands.py

```diff
@@ -78,6 +78,7 @@ def load_command_table(self, _):
         g.command('list', 'list_by_namespace')
         g.command('delete', 'delete')
         g.generic_update_command('update', custom_func_name='cli_sbqueue_update')
+        g.custom_command('exists', 'cli_queue_exists')
```

Comment:
https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md#non-standard-commands. 'exists` is ok. However, I'm just curious about it. Based on your implementation, the `show` command can do the same thing if the server return a good message. Is it necessary to add this command?